recipes/metplus/meta.yaml

```diff
@@ -0,0 +1,192 @@
+{% set version = "6.0.0" %}
+{% set met_version = "12.0.2" %}
+{% set ecbuild_version = "3.9.0" %}
+{% set eckit_version = "1.28.4" %}
+{% set atlas_version = "0.37.0" %}
+
+package:
+  name: metplus
+  version: {{ version }}
+
+source:
+  - url: https://github.com/dtcenter/METplus/archive/refs/tags/v{{ version }}.tar.gz
+    md5: a5612dfd025055707f4a45dff028e628
+    sha256: e9358aede2fd2abecd81806227de7b165d68fdf2fc9defcbba24df229461b155
+
+  - url: https://github.com/dtcenter/MET/archive/refs/tags/v{{ met_version }}.tar.gz
+    md5: 4b9dba80639a74f1a3ecb15357785603
+    sha256: 613c74cbe5a0afa87422e81a4cd70984238c1a0eaf1a395503486331223ede45
+    folder: MET
+
+  - url: https://downloads.sourceforge.net/project/gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/ghostscript-fonts-std-8.11.tar.gz
+    md5: 6865682b095f8c4500c54b285ff05ef6
+    sha256: 0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401
+    folder: gs-fonts
+
+  - url: https://github.com/ecmwf/ecbuild/archive/refs/tags/{{ ecbuild_version }}.tar.gz
+    md5: d2117212580160c24f62649643fa92e5
+    sha256: 8ad20169a7d917d6ac81a7ca0d1b11616e2aeb82c7782f6ae5b768603a3e000a
+    folder: ecbuild
+
+  - url: https://github.com/ecmwf/eckit/archive/refs/tags/{{ eckit_version }}.tar.gz
+    md5: 2f5299480ad43b53fa0df8482683968f
+    sha256: c6215efc5b6f190e9dd239e7784986f9d72381e70cb83b1067538240cc9b5c41
+    folder: eckit
+
+  - url: https://github.com/ecmwf/atlas/archive/refs/tags/{{ atlas_version }}.tar.gz
+    md5: 3d27a1b3273e06a21bbab1982278ad58
+    sha256: 26edab963e7f17bf4759924af4d3fb45582a64d1afd003d50a93ea7cf6f052a5
+    folder: atlas
+
+build:
+  skip: True  # [win]
+  number: 0
+  missing_dso_whitelist:
+    # eckit/atlas libraries are installed within this recipe
+    {% if osx %}
+    - libeckit.dylib
+    - libeckit_geometry.dylib
+    - libatlas.dylib
+    {% else %}
+    - libeckit.so
+    - libeckit_geometry.so
+    - libatlas.so
+    {% endif %}
+  entry_points:
+    - run_metplus.py = metplus.scripts.run_metplus:cli_main
+
+# Notes:
+# - GCC 12 on Linux and fftw are needed to install atlas
+# - libopenblas v0.3.28 is needed using 0.3.29 causes eckit to not find
+#  the library and it compiles with other libraries that are not included
+#  in this recipe.
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - libcxx
+    - cairo
+    - freetype
+    - gsl
+    - libnetcdf
+    - netcdf-cxx4
+    - libopenblas 0.3.28
+    - python 3.10
```

Comment:
> Note: Recipe must use Python 3.10 because the MET C++ code needs to be compiled with the same version of Python that will be installed.

This is normal for packages which are compiling python extensions. However, we have pinnings which handle this behavior already and build the package for multiple python versions. If your python package is only compatible with python 3.10, then pinning like this is probably fine, but if it can be built with other python versions, then you should instead use skip statements to skip python versions which are not compatible. e.g.

```yaml
build:
  skip: true  # [py<310]
```

Reply:
It looks like nceplibs-bufr requires python 3.10, so I wasn't able to test using other versions of python. Keeping the required version pinned to 3.10 for now.

Reply:
This should not be in `build`, though.